nsxt/resource_nsxt_policy_l2_vpn_session.go

```diff
@@ -0,0 +1,227 @@
+/* Copyright © 2022 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/vmware/vsphere-automation-sdk-go/runtime/protocol/client"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/infra/tier_0s/locale_services/l2vpn_services"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/model"
+)
+
+func resourceNsxtPolicyL2VPNSession() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceNsxtPolicyL2VPNSessionCreate,
+		Read:   resourceNsxtPolicyL2VPNSessionRead,
+		Update: resourceNsxtPolicyL2VPNSessionUpdate,
+		Delete: resourceNsxtPolicyL2VPNSessionDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"nsx_id":       getNsxIDSchema(),
+			"path":         getPathSchema(),
+			"display_name": getDisplayNameSchema(),
+			"description":  getDescriptionSchema(),
+			"revision":     getRevisionSchema(),
+			"tag":          getTagsSchema(),
+			"service_path": getPolicyPathSchema(true, true, "Policy path for L2 VPN service"),
+			"transport_tunnels": {
```

Comment:
Sorry if I'm missing something, but shouldn't there be more attributes here? I see some in both UI and the SDK under `L2VPNSession`

Reply:
yes you are right, I just added the mss and direction attribute for TcpMssClamping and also local/peer address and protocol for its TunnelEncapsulation